martin/src/pg/configurator.rs

```diff
@@ -420,6 +431,8 @@ mod tests {
             source_id_format: source_id_format.to_string(),
             schemas: schemas.map(|s| s.iter().map(|s| (*s).to_string()).collect()),
             id_columns: None,
+            clip_geom: None,
+            buffer: None,
```

Comment:
you need to do the same type of check as for `id_columns` for both of these, e.g.

```suggestion
            clip_geom: if item.clip_geom.is_some() && is_function {
                error!("Configuration parameter auto_publish.functions.clip_geom is not supported");
                None
            } else {
                item.clip_geom
            },
            buffer: if item.buffer.is_some() && is_function {
                error!("Configuration parameter auto_publish.functions.buffer is not supported");
                None
            } else {
                item.buffer
            },
```